spec/all_pages_spec.rb

```diff
@@ -51,6 +51,17 @@ def get_locale(path)
         end
       end
 
+      it 'has correctly configured system-status meta tag' do
+        doc.css('meta').each do |meta|
+          if meta[:name] == 'system-status'
+            puts(meta[:name])
+            puts('ok')
+            puts(meta[:content])
```

Comment:
We'll want to get rid of these debugging statements.

```suggestion
```

Reply:
those are removed